src/python_info.rs

```diff
@@ -49,16 +49,12 @@ fn get_python_binary(requested_python: &Option<String>) -> Result<std::path::Pat
     }
 
     let python3 = which::which("python3");
-    if python3.is_ok() {
-        return Ok(python3.unwrap());
+    if let Ok(p) = python3 {
+        return Ok(p);
     }
 
-    // Python3 may be called 'python', for instance on Windows
-    let res = which::which("python");
-    if res.is_err() {
-        return Err(Error::Other {
-            message: "Neither `python3` nor `python` fonud in PATH".to_string(),
-        });
-    }
-    Ok(res.unwrap())
+    let res = which::which("python").map_err(|_| Error::Other {
+        message: "Neither `python3` nor `python` found in PATH".to_string(),
+    })?;
+    Ok(res)
```

Comment:
You can just return the result instead of unwrapping and re-wrapping it, yes?

```
which::which("python").map_err(|_| Error::Other {
    message: "Neither `python3` nor `python` found in PATH".to_string(),
})
```